pkg/backend/cmd/server/server.go

```diff
@@ -537,6 +548,24 @@ func (s *server) Run(ctx context.Context, metrics metrics.Metrics, logger *zap.L
 		rpc.WithLogger(logger),
 	)
 	go autoOpsServer.Run()
+	// publicAPIService
+	publicAPIService := publicapi.NewPublicAPIService(
+		accountClient,
+		featureClient,
+		persistentRedisV3Cache,
```

Comment:
Please use `nonPersistentRedisV3Cache` instead.

```suggestion
		nonPersistentRedisV3Cache,
```